pavelib/tests.py

```diff
@@ -212,6 +212,26 @@ def coverage(options):
     call_task('diff_coverage', options=dict(options))
 
 
+@no_help
+@task
+@needs('pavelib.prereqs.install_prereqs')
+def combine_jenkins_coverage():
```

Comment:
Perhaps the coverage file would be something else to keep in the testeng-ci repo in the future?